fpgaconvnet/models/network/visualise.py

```diff
@@ -76,4 +79,67 @@ def plot_percentage_resource_per_layer_type(self, output_path=None):
     if output_path == None:
         plt.show()
 
-
+def visualise_partitions_nx(self, output_path):
```

Comment:
there is already a "visualise" function in Network.py, and I am not sure how they are different

Reply:
Actually, I created this one before seeing that there was another one doing the same thing.
In any case, the original Network visualise function depends on the visualise functions for each layer of the graph, which are not always implemented and are currently a bit buggy.
I kept this one as I need to check the partitions and how they are structured (mostly as a debugging tool).
We might drop this before merging, BUT we need to fix the other one in that case.